### PR TITLE
fix: Using `safeArrayPush(dest, source)` instead of `dest.push(...source)` (#506)

### PR DESCRIPTION
**Pull request by BanderasPRO**

### CODE REVIEW

File: packages/orama/tests/insert.test.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Index } from '../src/components/index.js'
 import { getInternalDocumentId } from '../src/components/internal-document-id-store.js'
 import { AnyDocument, create, insert, insertMultiple, search } from '../src/index.js'
 import dataset from './datasets/events.json' assert { type: 'json' }
+import { safeArrayPush } from "../src/utils"
```
Comment (allevo) [inline packages/orama/tests/insert.test.ts:7]: This is not needed. in the test, we can use `push` because the number of the objects here is not so high

Comment (allevo) [reply packages/orama/tests/insert.test.ts:7]: Also: the build fails for this reason.

**Comment by allevo** [comment]
LGTM when the CI passes.

File: packages/orama/tests/insert.test.ts (hunk 1 above)
Comment (BanderasPRO) [reply packages/orama/tests/insert.test.ts]: done

**Review by allevo — APPROVED** [review]
LGTM

**Review by H4ad — APPROVED** [review]
Awesome, thanks for the PR, and congratulations on your first PR!

**Comment by micheleriva** [comment]
@H4ad thanks for your support!

**Pull request merged**